libs/coin-modules/coin-canton/src/bridge/onboard.ts

```diff
@@ -110,6 +124,7 @@ export const buildOnboardAccount =
         await submitOnboarding(currency, publicKey, preparedTransaction, signature);
 
         const onboardedAccount = createOnboardedAccount(account, partyId, currency);
+        overrideAccountId(account, partyId, currency);
```

Comment:
Don't we do the same thing in createOnboardedAccount ? if it works now maybe we can update createOnboardedAccount ?